static_src/components/user_list.jsx

```diff
@@ -126,47 +126,49 @@ export default class UserList extends React.Component {
         { this.documentation }
         <ComplexList>
           { this.props.users.map((user) => {
-            let actions;
-            if (this.props.onRemove) {
-              let button = <span></span>;
-              if (this.props.currentUserAccess) {
-                if (this.props.userType === 'org_users') {
-                  buttonText = 'Remove User From Org';
-                } else if (this.props.userType === 'space_users') {
-                  buttonText = 'Remove All Space Roles';
+            if (!!user.username) {
```

Comment:
Keeping a map here is going to create a sparse array with `undefined`s where no JSX is returned. I would filter the list first on the `user.username` check. Doing that also provides the extra benefit of reducing the levels of nested `if` checks in the function!